libphobos/libdruntime/rt/dmain2.d

```diff
@@ -340,7 +340,15 @@ extern (C) int _d_run_main(int argc, char **argv, MainFunc mainFunc)
     version (CRuntime_Microsoft)
     {
         // enable full precision for reals
-        version (Win64)
+        version (GNU)
+        {
+            size_t fpu_cw;
+            asm { "fstcw %0" : "=m" (fpu_cw); }
+            fpu_cw |= 0b11_00_111111;  // 11: use 64 bit extended-precision
+                                       // 111111: mask all FP exceptions
+            asm { "fldcw %0" : "=m" (fpu_cw); }
```

Comment:
Hmm, this should be `: : "m" (fpu_cw)`.

Reply:
Ooops, you're right of course. I'll post an update soon.

Maybe I can also setup MinGW on CI now that we have some more time until 9.2. It should also be possible to get the tests to pass completely in that time frame.